packages/widget/src/context/wallet.ts

```diff
@@ -31,6 +34,12 @@ export enum WalletContextKeys {
   SUBSTRATE_WALLET = 'substrateWallet'
 }
 
+export type ParachainProviders = Map<number, ApiPromise>;
```

Comment:
Maybe add comment explaining number is parachain id. Or create `type ParachainId=number;`